src/main.rs

```diff
@@ -157,6 +181,7 @@ async fn root_get() -> Html<&'static str> {
 }
 
 async fn root_post(
```

Comment:
I would really like to see a test for this, but given the priority of this feature we should probably have a follow-up for that. Could you file an issue?